app/src/main/java/com/gathering/android/event/home/HomeFragment.kt

```diff
@@ -57,6 +57,11 @@ class HomeFragment : Fragment() {
             )
         )
         viewModel.onViewCreated()
+
+        binding.filterButton.setOnClickListener {
+            val dialog = FilterDialogFragment(viewModel)
+            dialog.show(parentFragmentManager, "filter_dialog")
```

Comment:
lets create a static const val in your `FilterDialogFragment` and put the tag there